lib/demos/material/vertical_divider_demo.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/gallery_localizations.dart';
+
+// BEGIN verDividerDemo
+
+class VerDividerDemo extends StatelessWidget {

```

Comment:
Instead of making a separate class and dart file for this demo, can you just use one class with an enum that sets it up differently depending on the value, like the other demos:
https://github.com/flutter/gallery/blob/master/lib/demos/material/button_demo.dart#L9